artemis/tests/auvergne_test.py

```diff
@@ -145,6 +145,92 @@ def test_project_coord_car_bike_with_dense_walking(self):
                      _min_car=0,
                      _min_bike=0)
 
+    def test_auvergne_01_with_direct_path_mode_bike(self):
+        """
+        Same reponse as test_auvergne_01
```

Comment:
minor :fr:-:gb: (same multiple times)
```suggestion
        Same response as test_auvergne_01
```